notebook/i18n/fr_FR/LC_MESSAGES/nbjs.po

```diff
@@ -381,7 +382,7 @@ msgstr "transformer en cellule de code"
 #: notebook/static/notebook/js/actions.js:373
 #: notebook/static/notebook/js/actions.js:374
 msgid "change cell to markdown"
-msgstr "transformer en cellule markdown"
+msgstr "transformer celllule en markdown"
```

Comment:
also 3 L now...

Reply:
That is my feeling, because « transformer cellule en markdown » suggests a transformation in cell content (like a conversion), which is wrong, to my understanding.

Plus the form you suggest is a bit grammatically clumsy (missing article), but maybe that is for clumsy-consistency with other messages of mine, I don't remember :-p.

No strong opinion on this though.